src/AWS.Deploy.CLI/Commands/TypeHints/DotnetBeanstalkPlatformArnCommand.cs

```diff
@@ -33,10 +33,16 @@ private async Task<List<PlatformSummary>> GetData()
             return await _awsResourceQueryer.GetElasticBeanstalkPlatformArns();
         }
 
-        public async Task<List<TypeHintResource>?> GetResources(Recommendation recommendation, OptionSettingItem optionSetting)
+        public async Task<TypeHintResourceTable> GetResources(Recommendation recommendation, OptionSettingItem optionSetting)
         {
             var platformArns = await GetData();
-            return platformArns.Select(x => new TypeHintResource(x.PlatformArn, $"{x.PlatformBranchName} v{x.PlatformVersion}")).ToList();
+
+            var resourceTable = new TypeHintResourceTable
+            {
+                Rows = platformArns.Select(x => new TypeHintResource(x.PlatformArn, $"{x.PlatformBranchName} v{x.PlatformVersion}")).ToList()
```

Comment:
Since we have columns support now you might as well put branch and version in separate columns.

Reply:
Added. I opted to keep the `$"{branch name} v{version}"` we have for the CLI, but we're now returning two columns for server mode.